codelabs/gke-base/install/enable_apis.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+bold() {
+  echo ". $(tput bold)" "$*" "$(tput sgr0)";
+}
+
+bold "Enabling Pub/Sub API..."
+
+gcloud services --project $PROJECT_ID enable pubsub.googleapis.com
+
+
+bold "Enabling Cloud Build API..."
+
+gcloud services --project $PROJECT_ID enable cloudbuild.googleapis.com
+
+
+bold "Enabling Kubernetes Engine API..."
+
+gcloud services --project $PROJECT_ID enable container.googleapis.com
```

Comment:
Do you think we'll need more APIs in other codelabs? If so, maybe we can drop a note to move this list of apis to the `properties` file in a future change.

Reply:
Done.